Content.Client/Construction/UI/ConstructionMenu.xaml

```diff
@@ -1,29 +1,35 @@
 <DefaultWindow xmlns="https://spacestation14.io">
     <BoxContainer Orientation="Horizontal" HorizontalExpand="True">
         <BoxContainer Orientation="Vertical" HorizontalExpand="True" SizeFlagsStretchRatio="0.4">
-            <BoxContainer Orientation="Horizontal" HorizontalExpand="True">
+            <BoxContainer Orientation="Horizontal" HorizontalExpand="True" Margin="0 0 0 5">
                 <LineEdit Name="SearchBar" PlaceHolder="Search" HorizontalExpand="True"/>
                 <OptionButton Name="Category" Access="Public" MinSize="130 0"/>
             </BoxContainer>
             <ItemList Name="Recipes" Access="Public" SelectMode="Single" VerticalExpand="True"/>
         </BoxContainer>
         <Control MinSize="10 0"/>
         <BoxContainer Orientation="Vertical" HorizontalExpand="True" SizeFlagsStretchRatio="0.6">
+            <Button Name="FavoriteButton" Visible="false" HorizontalExpand="False"
+                HorizontalAlignment="Right" Margin="0 0 0 15"/>
             <Control>
-                <BoxContainer Orientation="Horizontal" Align="Center">
-                    <TextureRect Name="TargetTexture" HorizontalAlignment="Right" Stretch="Keep"/>
-                    <Control MinSize="10 0"/>
-                    <BoxContainer Orientation="Vertical">
-                        <RichTextLabel Name="TargetName"/>
-                        <RichTextLabel Name="TargetDesc"/>
+                <BoxContainer Orientation="Vertical" HorizontalExpand="True" Margin="0 0 0 5">
+                    <BoxContainer Orientation="Horizontal" Align="Center">
+                        <TextureRect Name="TargetTexture" HorizontalAlignment="Right" Stretch="Keep"/>
+                        <Control MinSize="10 0"/>
```

Comment:
Should this dummy control just be a margin instead

Reply:
I don't know, this item was already there, and I didn't touch it. But I'll try to remove it.

Reply:
deleted  these controls